js/main.js

```diff
@@ -118,6 +124,7 @@ function viewSwap(view) {
   } else {
     $entries.setAttribute('class', '');
     $entryForm.setAttribute('class', 'hidden');
+    toggleNoEntries();
```

Comment:
This should be getting called outside your `if` or `else` but still inside the function